snapshot-cross-account-replicator/variables.tf

```diff
@@ -0,0 +1,26 @@
+variable "name" {
+  description = "Name of the setup"
+  type        = string
+}
+
+variable "rds_instance_ids" {
+  description = "List of IDs of the RDS instances to back up"
+  type        = list(string)
+}
+
+variable "snapshot_schedule_expression" {
+  description = "Snapshot frequency specified as a CloudWatch schedule expression. Can either be a `rate()` or `cron()` expression. Check the [AWS documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/ScheduledEvents.html#CronExpressions) on how to compose such expression."
+  type        = string
+  default     = "cron(0 */6 * * ? *)"
+}
+
+variable "retention_period" {
```

Comment:
So same retention priod is used for source and target accounts (although the target part still needs to be implemented in the script)?

Probably good enough for now.

Reply:
yes, it's the same retention period